docs/content/en/docs/install/install.md

```diff
@@ -80,44 +74,79 @@ as well as for upgrades.
 Some helpful hints:
 
 * Use the `--version <version>` flag on the
-  `helm upgrade --install` command line to specify a different Keptn version.
+  `helm upgrade --install` command to install a Keptn version
+  other than the latest.
+  For example,
+
+  ```shell
+  helm upgrade --install keptn keptn/keptn \
+   --version version klt-v0.8.2 \
+   -n keptn-lifecycle-toolkit-system --create-namespace --wait
+  ```
 
-* Use the following command sequence to see a list of available versions:
+  * Use the following command sequence to see a list of available versions:
+
+    ```shell
+    helm repo update
```

Comment:
it seems like the command to add the keptn helm repo got lost, so let's add this here before the `helm repo update`:

```
helm repo add keptn https://charts.lifecycle.keptn.sh
```

without this the charts won't be able to be installed

Reply:
Okay, I only showed the revised `helm update` command, not the full sequence since that is just a few lines above.  But I can put the whole sequence in.

Reply:
done